### ISSUE TITLE
Lost save

### ISSUE THREAD

**Issue by testestesttestestsets** [playest/hextml#13]
I had been using local saves for a few weeks on a project, but today when i clicked local it reloaded the page and now my previous saves are gone. Is there any way to get my old work back?

**Comment by playest** [comment]
Hello,

I'm sorry this happened to you. HexTML itself should not be able to delete your map. Refreshing the page should not delete anything.

Did you have a premium account and used the save on server feature by any chance?

Did you export it as html at some point and saved it on your hard drive?

Do you remember the exact name of your map? I will try to see if I can find something on the server.

**Comment by testestesttestestsets** [comment]
Hi, thanks for your reply. I do not have a Premium account. I do have some
old versions of it saved on my computer, but nothing recent.

Unfortunately I never changed the name so it was just called Map the Blank,
I just kept saving new versions under that name. I'm not sure if knowing
the date helps you search, but the most up to date versions that I lost
were saved on 7/26.

**Comment by playest** [comment]
Is there any keyword that could only be found in your map? Like the name of a place or a character maybe?

**Comment by testestesttestestsets** [comment]
Yeah, some region names I had labelled on the map that I made up would
be Magritta, Parsa, the Teuten Empire, and Zamar.

**Comment by playest** [comment]
So. It took me along time but I looked everywhere and I couldn't get back a copy of your map. I'm really sorry :(

It will not help you much now, but from now and for anyone that finds this github issue here is a post with some advice: https://www.patreon.com/posts/backup-your-maps-111642217

**Comment by testestesttestestsets** [comment]
Hi, it's all good, I eventually managed to recreate it. Appreciate your
help though.
